sections/method/topic_modelling.tex

```diff
@@ -15,6 +15,7 @@ \section{\acrlong{lda}}\label{sec:lda}
 	\label{fig:lda}
 \end{figure}
 \vejleder[inline]{beskriv hver plate i LDA}
```

Comment:
ville jeg mene at vi gør?

Reply:
jeg tror også han sagde at det var okay, men vi skal have et afsnit i bilag omkring variational learning for LDA